# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/readline/hotfix-install-no-rm.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

/bin/sh ../support/shlib-install -O linux-gnu -V t2 -d /usr/lib64 -b /usr/bin -i "/usr/src/t2-src/build/default-25-svn-generic-
x86-64-nocona-linux/TOOLCHAIN/native/wrapper/install -c -m 644" libhistory.so.8.3
bash: error while loading shared libraries: libhistory.so.8: cannot open shared object file: No such file or directory
chmod: cannot access '/usr/lib64/libhistory.so.8.3': No such file or directory

--- readline-8.3/support/shlib-install.vanilla	2026-02-07 13:14:29.787069396 +0100
+++ readline-8.3/support/shlib-install	2026-02-07 13:22:05.086581272 +0100
@@ -44,7 +44,7 @@
 OLDSUFF=old
 MV=mv
 RM="rm -f"
-LN="ln -s"
+LN="ln -sf"
 
 # pre-install
 
@@ -120,13 +120,13 @@
 case "$host_os-$host_vendor" in
 *linux*|freebsd*|dragonfly*)
 	# libname.so.M -> libname.so.M.N
-	${echo} ${RM} ${INSTALLDIR}/$LINK2
+	#${echo} ${RM} ${INSTALLDIR}/$LINK2
 	if [ -z "$uninstall" ]; then
 		eval $INSTALL_LINK2
 	fi
 
 	# libname.so -> libname.so.M
-	${echo} ${RM} ${INSTALLDIR}/$LINK1
+	#${echo} ${RM} ${INSTALLDIR}/$LINK1
 	if [ -z "$uninstall" ]; then
 		${echo} cd $INSTALLDIR && ${echo} ${LN} $LINK2 $LINK1
 	fi
